hw/bsp/spresense/board.mk

```diff
@@ -70,4 +70,5 @@ $(BUILD)/$(PROJECT).spk: $(MKSPK)
 
 # flash
 flash: $(BUILD)/$(PROJECT).spk
-	@$(TOP)/hw/mcu/sony/cxd56/tools/flash_writer.py -s -c $(SERIAL) -d -b 115200 -n $<
+	@echo FLASH $<
+	@$(PYTHON) $(TOP)/hw/mcu/sony/cxd56/tools/flash_writer.py -s -c $(SERIAL) -d -b 115200 -n $<
```

Comment:
Is adding $(PYTHON) necessary here?

Reply:
yeah I got following issue without explicitly invoked with python under ubuntu 20.04

```
/usr/bin/env: ‘python3\r’: No such file or directory
```

Reply:
Hmm.. I am also using Ubuntu 20.04 and I don't have this issue. Have you upgraded your Ubuntu from a previous version or have you installed a new Ubuntu 20.04?

If you change `PYTHON = python` to `PYTHON = python3` will it work for you?